app/services/arcgis_api/mock/fixtures.rb

```diff
@@ -14,7 +14,7 @@ def self.request_suggestions_error_html
       end
 
       def self.request_candidates_response
-        load_response_fixture('request_candidates_response.json')
```

Comment:
Is the `request_candidates_response.json` file unused now? If so, should it be removed?